trade-finance-manager-api/src/graphql/helpers/amendment.helpers.api-test.js

```diff
@@ -0,0 +1,292 @@
+const amendmentHelpers = require('./amendment.helpers');
+const { CURRENCY } = require('../../constants/currency.constant');
+const { FACILITY_TYPE } = require('../../constants/facilities');
+const { AMENDMENT_UW_DECISION, AMENDMENT_BANK_DECISION } = require('../../constants/deals');
+const api = require('../../v1/api');
+
+describe('amendmentChangeValueExportCurrency()', () => {
+  const amendment = { currency: CURRENCY.GBP };
+
+  it('should return a string with currency and value to 2 d.p', () => {
+    amendment.value = 25000;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = `${CURRENCY.GBP} 25,000.00`;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return a string with currency and value to 2 d.p even if different currency', () => {
+    amendment.value = 25000;
+    amendment.currency = 'JPY';
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = 'JPY 25,000.00';
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no value', () => {
+    amendment.value = null;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no currency', () => {
+    amendment.value = 25000;
+    amendment.currency = null;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+
+  it('should return null if no currency or no value', () => {
+    amendment.value = null;
+    amendment.currency = null;
+
+    const result = amendmentHelpers.amendmentChangeValueExportCurrency(amendment);
+
+    const expected = null;
+
+    expect(result).toEqual(expected);
+  });
+});
+
+describe('calculateNewFacilityValue()', () => {
+  const amendment = { currency: CURRENCY.GBP };
+
+  it('should return the same number since in GBP', () => {
+    amendment.value = 25000;
+
+    const result = amendmentHelpers.calculateNewFacilityValue(null, amendment);
+
```

Comment:
we don't need these spaces between lines

Reply:
removed